libs/pika/schedulers/include/pika/schedulers/queue_holder_thread.hpp

```diff
@@ -464,7 +468,7 @@ namespace pika { namespace threads { namespace policies {
             threads::thread_id_ref_type& tid, threads::thread_init_data& data)
         {
             PIKA_ASSERT(data.stacksize >= thread_stacksize::minimal);
-            PIKA_ASSERT(data.stacksize <= thread_stacksize::maximal);
+            PIKA_ASSERT(data.stacksize <= thread_stacksize::nostack);
```

Comment:
Do you think it would make sense to reorderd the stacksizes so that `nostack == minimal`?

Reply:
it probably would make sense, but that ought to be done in a separate PR, so I will leave this 'as is' for now

Reply:
https://github.com/pika-org/pika/issues/213